src/components/screens/PointOfInterest.js

```diff
@@ -154,6 +148,20 @@ export const PointOfInterest = ({ data, navigation }) => {
           </View>
         )}
 
+        {!hideMap && !!latitude && !!longitude && (
+          <View>
+            <TitleContainer>
+              <Title accessibilityLabel={`${texts.pointOfInterest.location} (Überschrift)`}>
+                {texts.pointOfInterest.location}
+              </Title>
+            </TitleContainer>
+            <WebViewMap locations={[{ position: { lat: latitude, lng: longitude } }]}
+              style={{ height: normalize(200) }}
```

Comment:
a 16:9 ratio depending on the device would fit better for different device sizes. see my comment other comment on this at `LocationOverview.tsx`

Reply:
see above (done in c57107eb38b453d5499ca788e7a5cab02becb237)